sdk/core/azure-core-tracing-opentelemetry/inc/azure/core/tracing.hpp

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
The header should be named `opentelemetry.hpp`, and it should be in `azure/core/tracing/`